coala_quickstart/generation/Bears.py

```diff
@@ -37,7 +35,7 @@ def filter_relevant_bears(used_languages,
         A dict with language name as key and bear classes as value.
     """
     args = arg_parser.parse_args() if arg_parser else None
-    log_printer = LogPrinter(NullPrinter())
+    log_printer = None
```

Comment:
Initializing it to `None` everywhere can be confusing for someone who is not aware of this deprecation issue. IMO, it's better to pass it as keyword argument like `log_printer=None` while invoking a function (like [here](https://github.com/coala/coala-quickstart/pull/191/files#diff-f6f9ce615c88e726f659111f8e2d547aR43).